app/src/features/onboarding/components/persona/components/IndustryInput/index.scss

```diff
@@ -0,0 +1,39 @@
+.industry-input {
+  width: 100% !important;
+  margin-top: 4px !important;
+
+  & .ant-select-selector {
+    background: #2b2b2b !important;
+    border: 1px solid var(--neutrals-gray-600) !important;
```

Comment:
Please check if the same color is present in the new tokens and replace it. Same for other variables and colors used here as well.